adapters/MolocoAdapter/MolocoAdapter/MolocoSdkImpl.swift

```diff
@@ -74,3 +74,12 @@ extension MolocoSdkImpl: MolocoBannerFactory {
   }
 
 }
+
+// MARK: - MolocoBidTokenGetter
+
+extension MolocoSdkImpl: MolocoBidTokenGetter {
+
+  func getBidToken(completion: @escaping (String?, (any Error)?) -> Void) {
+    Moloco.shared.getBidToken(completion: completion)
```

Comment:
We also return an error for BidToken 

`public func getBidToken(completion: @escaping (_ bidToken: String?, _ error: Error?) -> Void)`

Reply:
Sounds good. MolocoBidTokenGetter is just a wrapper around Moloco SDK's getBidToken function. The error is handled in the collectSignals() implementation within MolocoMediationAdapter.